sdks/python/apache_beam/runners/interactive/cache_manager.py

```diff
@@ -30,6 +30,14 @@
 from apache_beam.io import filesystems
 from apache_beam.transforms import combiners
 
+try:                    # Python 3
```

Comment:
Is there a reason not to use 
```
from future.moves.urllib.parse import quote
from future.moves.urllib.parse import unquote
```
for consistency with dataflow_runner.py?

Reply:
The input of the `unquote()` call here is a `bytes` object, while it is a `str` in the dataflow_runner.

Reply:
`urllib.parse.unquote` requires a `str` on Python 3, so this will still not work if we give it a `bytes` object?

Reply:
Ah, nevermind, `urllib.parse.unquote_to_bytes` can accept `string` or `bytes`.

Reply:
I also couldn't just use `urllib.parse.unquote_to_bytes` on both Python 2 and 3, since it returnes a `future.builtins.newbytes` instance on Python 2.